.copr/Makefile

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Could you please add comment with link to this documentation: https://docs.pagure.org/copr.copr/user_documentation.html explaining why this `Makefile` exists here.

Reply:
Good idea. I'll document it here, as well as add general documentation about how
to make SRPMs to CONTRIBUTING.